.github/workflows/deploy-container.yml

```diff
@@ -19,4 +19,20 @@ jobs:
           registry: ghcr.io
           githubOrg: EOS-uiux-Solutions
           username: ${{ secrets.GHCR_USERNAME }}
+          password: ${{ secrets.GHCR_PASSWORD }}
+
+  publish-test-image:
```

Comment:
@SundeepChand so If I understand this correctly, we're now publishing a `test image` with some dummy date, right?